ext/skywalking/dashboard/dolphinscheduler.yml

```diff
@@ -0,0 +1,284 @@
+                  "unit":"%",
+                  "aggregation":"/",
+                  "aggregationNum":"100"
+                },
+                {
+                  "width":3,
+                  "title":"Take Load",
+                  "height":350,
+                  "entityType":"Endpoint",
+                  "independentSelector":true,
+                  "metricType":"REGULAR_VALUE",
+                  "metricName":"endpoint_cpm",
+                  "queryMetricType":"readMetricsValues",
+                  "chartType":"ChartLine",
+                  "currentService":"dolphinscheduler::master-server",
+                  "currentEndpoint":"masetr/queue/take"
+                },
+                {
+                  "width":3,
+                  "title":"Take Avg Time",
+                  "height":350,
+                  "entityType":"Endpoint",
+                  "independentSelector":true,
+                  "metricType":"REGULAR_VALUE",
+                  "metricName":"endpoint_avg",
+                  "queryMetricType":"readMetricsValues",
+                  "chartType":"ChartLine",
+                  "currentService":"dolphinscheduler::master-server",
+                  "currentEndpoint":"masetr/queue/take",
+                  "unit":"ms"
+                },
+                {
+                  "width":3,
+                  "title":"Take Time Percentile",
+                  "height":350,
+                  "entityType":"Endpoint",
+                  "independentSelector":true,
+                  "metricType":"LABELED_VALUE",
+                  "metricName":"endpoint_percentile",
+                  "queryMetricType":"readLabeledMetricsValues",
+                  "chartType":"ChartLine",
+                  "metricLabels":"P50, P75, P90, P95, P99",
+                  "labelsIndex":"0, 1, 2, 3, 4",
+                  "currentService":"dolphinscheduler::master-server",
+                  "currentEndpoint":"masetr/queue/take",
+                  "unit":"ms"
+                },
+                {
+                  "width":3,
+                  "title":"Take Successful Rate",
+                  "height":350,
+                  "entityType":"Endpoint",
+                  "independentSelector":true,
+                  "metricType":"REGULAR_VALUE",
+                  "metricName":"endpoint_sla",
+                  "queryMetricType":"readMetricsValues",
+                  "chartType":"ChartLine",
+                  "currentService":"dolphinscheduler::master-server",
+                  "currentEndpoint":"masetr/queue/take",
+                  "unit":"%",
+                  "aggregation":"/",
+                  "aggregationNum":"100"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    # Activated as the DASHBOARD type, makes this templates added into the UI page automatically.
+    # False means providing a basic template, user needs to add it manually.
+    activated: false
```

Comment:
I recommend to use `true`, because we have asked users to add this dashboard setting to the SkyWalking's folder manually. Once they added, they should want to see this shows up on the browser.

Reply:
👌